### ISSUE TITLE
Circuitry Mount Assembly: Where to insert the screw inserts is unclear

### ISSUE THREAD

**Issue by TrueshotBarrage** [personalrobotics/ada_mouth#1]
**Summary: Steps 5-8 for the circuitry mount assembly documentation are unclear and are consequently hard to follow when building the mouth model.**

> Step 5: Use a soldering iron to gently push an M2 screw insert into **each of the 16 holes** on the back of the circuitry mount.

First, the orientation of the circuitry mount should be specified. "Front" or "back" do not make sense without a reference frame. Regardless, there doesn't seem to be any set of 16 holes on any of the surfaces in the 3d STL file. As such, it is impossible to follow the rest of the instructions in the section, as the location of the strain gauges is unclear.

**Comment by TrueshotBarrage** [comment]
![Strain gauge board](https://github.com/personalrobotics/ada_mouth/wiki/images/resolder_stain_gauge.png)

Judging from this image, there seems to be a different/additional part that is not included in the documentation. The [mouth_circuitry_mount.stl](https://github.com/personalrobotics/ada_mouth/blob/master/mouth_hw/STL/mouth_circuitry_mount.stl) only includes the lower half of what is displayed in the image.

**Comment by palanc** [comment]
Thanks for pointing this out, we had accidentally posted an older version of that part. It is updated now.
